### ISSUE TITLE
Just a thank you.

### ISSUE THREAD

**Issue by jordanwilliams1** [thorsten-gehrig/alexa-remote-control#98]
Just wanted to say thank you for making such an amazing script :) I've made some pretty cool stuff. Thanks 👍

**Comment by W7RZL** [comment]
Indeed! Thanks a bunch - this addresses several shortcomings in the Alexa Routine/Skills integration.

**Comment by von-Chaps** [comment]
Yep, many thanks. Very useful.

**Comment by FROeHlyEisvogel** [comment]
Really cool script. I'm using it with pimatic home automation to announce me when the bath window is open for more than 15 min.

**Comment by petrasl1976** [comment]
This is what i was looking for since i bought Alexa :) The first thing i missed on the first day was to pronounce random phrase from my list on some trigger. Toons of "skills" can do similar stuff, but none of them to say phrase without any "Hello, this is..." or "Do you want more?". With this script i can make Alexa speak as i want! Thanks, it is really great job!

**Comment by mkozjak** [comment]
This is an amazing piece of work! Making Alexa play the songs I need is working at the 50% success rate. This script makes it work 100%. :)
